app/routes/admin/__paths/customPayPlan.tsx

```diff
@@ -0,0 +1,53 @@
+import { Button, Container, Select } from "@pokt-foundation/pocket-blocks"
+import { LinksFunction } from "@remix-run/node"
+import { useFetcher } from "@remix-run/react"
+import { useState } from "react"
+import Card, { links as CardLinks } from "~/components/shared/Card"
+import TextInput, { links as TextInputLinks } from "~/components/shared/TextInput"
+import { PayPlanType } from "~/models/portal/sdk"
+
+export const links: LinksFunction = () => [...CardLinks(), ...TextInputLinks()]
+
+export default function CustomPayPlan() {
+  const [selectedPlanType, setSelectedPlanType] = useState<string | null>(null)
+  const updatePlanFetcher = useFetcher()
```

Comment:
I think it would be great to spread the components abstracted from hooks to keep the consistency between lowercase for HTML and Capital Case for React components.

```suggestion
  const { Form, ...updatePlanFetcher } = useFetcher()
```